src/main/java/com/github/manolo8/darkbot/core/objects/facades/DispatchGateProxy.java

```diff
@@ -0,0 +1,14 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+import eu.darkbot.api.API;
+import lombok.Getter;
+
+@Getter
+public class DispatchGateProxy extends Updatable implements API.Singleton {
```

Comment:
is this used for anything?

Reply:
nothing is using this at the moment, commented out for now